l10n_it_statutory_financial_statements/data/cee_groups.xml

```diff
@@ -0,0 +1,947 @@
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.18.D</field>
+            <field name="name">Di strumenti finanziari derivati</field>
+            <field name="parent_id" ref="account_group_3d18"/>
+        </record>
+
+        <record id="account_group_3d19" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.19</field>
+            <field name="name">Svalutazioni</field>
+            <field name="parent_id" ref="account_group_3d"/>
+        </record>
+
+        <record id="account_group_3d19a" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.19.A</field>
+            <field name="name">Di partecipazioni</field>
+            <field name="parent_id" ref="account_group_3d19"/>
+        </record>
+        <record id="account_group_3d19b" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.19.B</field>
+            <field name="name">Di immobilizzazioni finanziarie che non costituiscono partecipazioni</field>
+            <field name="parent_id" ref="account_group_3d19"/>
+        </record>
+        <record id="account_group_3d19c" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.19.C</field>
+            <field name="name">Di titoli iscritti all’attivo circolante che non costituiscono partecipazioni</field>
+            <field name="parent_id" ref="account_group_3d19"/>
+        </record>
+        <record id="account_group_3d19d" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.D.19.D</field>
+            <field name="name">Di strumenti finanziari derivati</field>
+            <field name="parent_id" ref="account_group_3d19"/>
+        </record>
+
+        <record id="account_group_3e" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.E</field>
+            <field name="name">Proventi e oneri straordinari</field>
+            <field name="parent_id" ref="account_group_3"/>
+        </record>
+
+        <record id="account_group_3e20" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.E.20</field>
+            <field name="name">Imposte sul reddito dell’esercizio, correnti, differite, anticipate</field>
+            <field name="parent_id" ref="account_group_3e"/>
+        </record>
+        <record id="account_group_3e21" model="account.group">
+            <field name="cee_group" eval="True"/>
+            <field name="code_prefix">3.E.21</field>
+            <field name="name">Utile (perdita) dell’esercizio</field>
+            <field name="parent_id" ref="account_group_3e"/>
+        </record>
+    </data>
+</odoo>
```

Comment:
Se per caso compaiono nel pot andrebbero in inglese.

Reply:
Le voci del bilancio sono definite nel codice civile e non le tradurrei

Reply:
:+1: